sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts

```diff
@@ -230,7 +235,8 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
     scopes: string[],
     options: CredentialFlowGetTokenOptions = {}
   ): Promise<AccessToken> {
-    options.correlationId = options?.correlationId || this.generateUuid();
+    options.correlationId =
+      options?.correlationId || this.defaultCorrelationId || this.generateUuid();
```

Comment:
Looks like we generate the uuid if nothing is present in this file that is specific for node flows. How is it that we don't do something similar for browser flows?

Reply:
I think it's safe to assume that empty string isn't a valid correlationId.

Reply:
MSAL generates one for us anyway, but I needed this to test this feature. Now that we have a way to set a default correlation ID we can remove `generateUuid` from here, since MSAL will do it for us... though I'm hesitant from leaving this to MSAL, since it's a first class feature for us.

Here's how MSAL generates a correlation Id if it's not provided: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/fd0dcd559af8ab0e04dec3c30cf3d4e122d6057d/lib/msal-common/src/client/AuthorizationCodeClient.ts#L214 they already assume that an empty string is not a valid correlation Id.